Library/Homebrew/cmd/list.rb

```diff
@@ -129,9 +129,9 @@ def list
         safe_system "ls", *ls_args, Cask::Caskroom.path
       end
     elsif args.verbose? || !$stdout.tty?
-      system_command! "find", args: args.named.to_kegs.map(&:to_s) + %w[-not -type d -print], print_stdout: true
+      system_command! "find", args: args.named.to_default_kegs.map(&:to_s) + %w[-not -type d -print], print_stdout: true
```

Comment:
Should `brew list <formula>` list items in each keg for that formula? I haven't tested so maybe it doesn't work but I wonder if it should stay as `to_kegs`

Reply:
In its current state (i.e., before any changed made by this PR), it only lists items in the default keg (the keg that is linked or would be linked if `brew link <formula>` was called).

The change above reflects the renaming of `to_kegs` to `to_default_kegs`.

I think that it's reasonable to have `brew list <formula>` list items in the default keg by default. However, it does seem a tad annoying that you can only list items in the default keg in the same way that you can only link the default keg. I think the best approach here would be to open a separate PR for this implementing the same behavior for `brew list` that we decide to implement for `brew link` (assuming changes are made).

Reply:
👍 I'm good with that

Reply:
> I think that it's reasonable to have `brew list <formula>` only list items in the default keg by default.

Agreed.